protocol/src/lib/create-loyalty-program.ts

```diff
@@ -0,0 +1,93 @@
+import { generateSigner, KeypairSigner, PublicKey } from '@metaplex-foundation/umi'
+import { createCollection, CreateCollectionArgsPlugin, PluginAuthority } from '@metaplex-foundation/mpl-core'
+import { ATTRIBUTE_KEYS, PLUGIN_TYPES } from '@/lib/constants'
+import { VerxioContext } from '@/types/verxio-context'
+import { toBase58 } from '@/utils/to-base58'
+import { LoyaltyProgramTier } from '@/types/loyalty-program-tier'
+import { assertValidContext } from '@/utils/assert-valid-context'
+
+export interface CreateLoyaltyProgramConfig {
```

Comment:
Now that we have a proper file for this function, it makes more sense to have the interface sit right next to it.